modules/hcp-ibm-z-zvm-agents.adoc

```diff
@@ -31,6 +31,19 @@ ai.ip_cfg_override=1 \// <4>
 <1> For the `coreos.live.rootfs_url` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are starting. Only HTTP and HTTPS protocols are supported.
 <2> For the `ip` parameter, manually assign the IP address, as described in _Installing a cluster with z/VM on {ibm-z-title} and {ibm-linuxone-title}_.
 <3> For installations on DASD-type disks, use `rd.dasd` to specify the DASD where Red Hat Enterprise Linux CoreOS (RHCOS) is to be installed. For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where RHCOS is to be installed.
++
+[NOTE]
+====
+For FCP multipath configurations, provide two disks instead of one.
+====
++
+.Example 
+[source,yaml]
+----
+rd.zfcp=<adapter1>,<wwpn1>,<lun1> \
+rd.zfcp=<adapter2>,<wwpn2>,<lun2>
+----
++
 <4> Specify this parameter when you use an Open Systems Adapter (OSA) or HiperSockets.
```

Comment:
```suggestion
[NOTE]
====
For FCP multipath configurations, provide two disks instead of one. For example:

[source,yaml]
----
rd.zfcp=<adapter1>,<wwpn1>,<lun1> \
rd.zfcp=<adapter2>,<wwpn2>,<lun2>
----
====
+
<4> Specify this parameter when you use an Open Systems Adapter (OSA) or HiperSockets.
```

Please ensure this renders fine. Let's see if check passes after file is updated with this change.